Kentico.Kontent.Management/Models/AssetRenditions/AssetRenditionCreateModel.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
+
+namespace Kentico.Kontent.Management.Models.AssetRenditions
+{
+    public class AssetRenditionCreateModel
+    {
+        [JsonProperty("external_id", Required = Required.Always)]
```

Comment:
Note: We don't use `Required` consistently, I have to think about if we want to use it at all because it just throws an obscure exception instead of throwing an API exception.

Reply:
Should I remove it from the models I added?